core/refinerycms-core.gemspec

```diff
@@ -2,7 +2,7 @@
 require File.expand_path('../../core/lib/refinery/version', __FILE__)
 
 version = Refinery::Version.to_s
-rails_version = ['>= 4.2.3', '< 5.0']
+rails_version = ['>= 5.0.0', '< 5.1']
```

Comment:
At this time no.
I've send you some PRs/Issues on Globalize, Decoratos and Seo_meta gems. 
After we have 37 tests to fix, nothing too hard IMHO. 

Should we start using Appraisals?